app/src/features/datasets/DatasetPage.tsx

```diff
@@ -162,7 +353,7 @@ const DatasetPage: React.FC<React.PropsWithChildren> = () => {
     <Box>
       <Paper square elevation={0} className={classes.datasetTitleContainer}>
         <Container maxWidth="xl">
-          <Typography variant="h1">{datasetDataLoader.data['eml:eml'].dataset.title}</Typography>
+          <RenderWithHandlebars dataset={datasetDataLoader} rawTemplate={rawTemplate as string}></RenderWithHandlebars>
```

Comment:
just checking: is the `as string` still needed?

Reply:
super nit: can remove the closing tag, and just add the slash to the opening tag: `<renderStuff />`